Source/items.cpp

```diff
@@ -3509,10 +3509,11 @@ void CreateTypeItem(Point position, bool onlygood, ItemType itemType, int imisc,
 	SetupBaseItem(position, idx, onlygood, sendmsg, delta, spawn);
 }
 
-void RecreateItem(const Player &player, Item &item, _item_indexes idx, uint16_t icreateinfo, uint32_t iseed, int ivalue, bool isHellfire)
+void RecreateItem(const Player &player, Item &item, _item_indexes idx, uint16_t icreateinfo, uint32_t iseed, int ivalue, uint32_t dwBuff)
 {
 	bool tmpIsHellfire = gbIsHellfire;
-	gbIsHellfire = isHellfire;
+	item.dwBuff = dwBuff;
+	gbIsHellfire = item.dwBuff & CF_HELLFIRE;

```

Comment:
`gbIsHellfire` is a boolean, so we should be assigning the result of a conditional statement instead of implicitly converting from an integer.
```suggestion
	gbIsHellfire = (item.dwBuff & CF_HELLFIRE) != 0;
```

Reply:
Yes, this is in your style guide. Thanks, and sorry about that.